### DISCUSSION TITLE
Newly added Twitter blogs only downloading a limited amount of posts

### DISCUSSION THREAD

**Discussion by Technotron21** [TumblThreeApp/TumblThree#429] [General]
I've found today that every new blog I've added to the application only seems to grab the last several hundred posts made by the user, as the application displays the amount of posts that can be downloaded going up to the 4 or 5 digits, before only deciding to grab around numbers in the 3 digits. I've only noticed this happening after the last update, so I'm not entirely sure if something got messed up there, or if it happened to coincide with further screw ups within the Twitter API. There is a possibility these specific accounts might've recently set it up to where they delete older posts past a certain date, but I doubt it with the crawler still being able to pick up on all these posts. Would be curious to hear back on this.

**Comment by thomas694** [comment]
Not sure, but there have been some hiccups recently.
<br>

> coincide with further screw ups within the Twitter API

[their announcement](https://github.com/TumblThreeApp/TumblThree/discussions/430)

**Reply by Technotron21** [reply]
I saw that right after. What a great shame. This post from gallery-dl (https://github.com/mikf/gallery-dl/issues/3601) does suggest it isn't likely to fully shut off soon, but I am now seeing something is in effect if age restricted accounts are once again flagged as being unavailable (no VPN to hamper things, either). So I guess the only thing to do at this point is properly implement some authentication thing while things are able to be crawled, because I'm sensing that part of it will be permanent sooner or later if g-dl has a workaround for it.

Edit: the age restricted accounts thing fixed itself very quickly, so probably a result of the ongoing changes.
